esmvaltool/cmorizers/obs/cmorize_obs_esacci_lst.py

```diff
@@ -0,0 +1,154 @@
+"""ESMValTool CMORizer for ESACCI-LST data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   On CEDA-JASMIN
+   /group_workspaces/jasmin2/esacci_lst/public
+
+Download and processing instructions
```

Comment:
also it'd be nice to say that one needs to register for the JASMIN group workspace, otherwise the data is unavailable, registration can happen at https://accounts.jasmin.ac.uk/services/group_workspaces/esacci_lst/

Reply:
Added some info on the registration! Thank for catching this.